lms/models/key.py

```diff
@@ -0,0 +1,41 @@
+import json
+from uuid import uuid4
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import UUID
+
+from lms.db import BASE
+from lms.models import CreatedUpdatedMixin
+
+
+class Key(CreatedUpdatedMixin, BASE):
```

Comment:
Is this a `JSONWebKey`? Key alone is quite vague.

Reply:
Went for RSAKey